tests/functional/u/unreachable.py

```diff
@@ -32,3 +33,15 @@ def func6():
     return
     yield
     print("unreachable")  # [unreachable]
+
+def func7():
+    sys.exit(1)
+    var = 2 + 2  # [unreachable]
+    print(var)
+
```

Comment:
```suggestion
class FalseExit:
    def exit():
        print("False positive not sys.exit")

def func_false_exit():
    sys  = FalseExit()
    sys.exit(1)
    var = 2 + 2
    print(var)
```
Following @jacobtylerwalls comment.